runtime/onert/core/src/util/shapeinf/Reshape.cc

```diff
@@ -109,17 +109,18 @@ void DynamicInferer::visit(const ir::operation::Reshape &op)
   if ((!input->is_dynamic()) && (!output->is_dynamic()))
     return;
 
-  // from op, access the buffer of second input to read new shape
-  auto new_shape_ind = op.getInputs().at(ir::operation::Reshape::Input::SHAPE);
-  auto &new_shape_op = _operands.at(new_shape_ind);
+  // input total_size;
+  size_t input_total_size = 1;
+  size_t rank = input->num_dimensions();
 
-  // if shape is from Const, TFLC put the shape of output into tensor
-  if (new_shape_op.isConstant())
+  for (size_t d = 0; d < rank; d++)
   {
-    // no change on output shape
-    return;
+    input_total_size *= input->dimension(d);
```

Comment:
@YiHyunJin You may use `getNumberOfElements`.

Reply:
getNumberOfElements is defined in OperationUtils.h. However, the relative path up to OperationUtils.h is complicated. like this..
`
"../../../../backend/cpu/ops/OperationUtils.h"
`
How can I use it simply?